src/main/java/org/folio/rs/domain/entity/Configuration.java

```diff
@@ -3,8 +3,8 @@
 import java.time.LocalDateTime;
 import java.util.UUID;
 
-import javax.persistence.*;
-import javax.validation.constraints.NotNull;
+import jakarta.persistence.*;
```

Comment:
you should use single imports

Reply:
Why?

Reply:
https://stackoverflow.com/questions/25260952/what-is-better-import-one-by-one-or
and .* imports are not allowed in most projects and maven-checkstyle-plugin which are not enabled in this module

Reply:
I think just changing one class's imports wouldn't make much sense, even if all classes were changed and if the IDEA code style isn't configured to do it automatically, then it would be problematic to track future imports. So, better team will decide.